src/context.ts

```diff
@@ -20,7 +20,10 @@ export interface Context<RequestParamKeyType extends string = string, E = Env> {
   header: (name: string, value: string) => void
   status: (status: StatusCode) => void
   set: (key: string, value: any) => void
```

Comment:
Set should probably also attempt to enforce the type of `value` if it's specified on `ContextVariableMap` for extra type safety.

Reply:
Thanks!
Fixed in 1852968.